DataFormats/Detectors/FIT/FDD/include/DataFormatsFDD/LookUpTable.h

```diff
@@ -214,21 +214,16 @@ class SingleLUT : public LUT
 
  public:
   static constexpr char sDetectorName[] = "FDD";
+  static constexpr char sDefaultCCDBpath[] = "http://ccdb-test.cern.ch:8080/";
+  static constexpr char sDefaultLUTpath[] = "FDD/LookUpTable";
```

Comment:
We are switching to 3-level CCDB directories, perhaps you can directly change them to e.g. `FDD/Config/LookUpTable` ?

Reply:
This is old CCDB directory, but target `FDD/Config/LookUpTable` is configurable via `--lut-path FDD/Config/LookUpTable --ccdb-path http://o2-ccdb.internal/` for raw2digit and digit2raw. I can upload new 3-level dir into `http://ccdb-test.cern.ch:8080/` (`http://o2-ccdb.internal/` unfortunately is unable for me right now, I'm discussion this situation with Chiara)